covid19-etl/etl/ccmap.py

```diff
@@ -0,0 +1,185 @@
+            ("Staffed All Beds [Per 1000 Elderly (65+)]", ("summary_report", "staffed_all_beds_per_1000_gtr_65", float)),
+            ("Staffed ICU Beds [Per 1000 People]", ("summary_report", "staffed_icu_beds_per_1000", float)),
+            ("Staffed ICU Beds [Per 1000 Adults (20+)]", ("summary_report", "staffed_icu_beds_per_1000_gtr_20", float)),
+            ("Staffed ICU Beds [Per 1000 Elderly (65+)]", ("summary_report", "staffed_icu_beds_per_1000_gtr_65", float)),
+            ("Licensed All Beds [Per 1000 People]", ("summary_report", "licensed_all_beds_per_1000", float)),
+            ("Licensed All Beds [Per 1000 Adults (20+)]", ("summary_report", "licensed_all_beds_per_1000_gtr_20", float)),
+            ("Licensed All Beds [Per 1000 Elderly (65+)]", ("summary_report", "licensed_all_beds_per_1000_gtr_65", float)),
+            ("Estimated No. Full-Featured Mechanical Ventilators (2010 study estimate)", ("summary_report", "estimated_full_mech_ventilators", int)),
+            ("Estimated No. Full-Featured Mechanical Ventilators per 100,000 Population (2010 study estimate)", ("summary_report", "estimated_full_mech_ventilators_per_100000", float)),
+            ("Estimated No. Pediatrics-Capable Full-Feature Mechanical Ventilators (2010 study estimate)", ("summary_report", "estimated_full_mech_pediatric_ventilators", int)),
+            ("Estimated No. Full-Feature Mechanical Ventilators, Pediatrics Capable per 100,000 Population <14 y (2010 study estimate)", ("summary_report", "estimated_full_mech_pediatric_ventilators_per_100000", float)),
+        ]
+
+        self.headers_mapping = {
+            "county": {field: (k, mapping) for k, (field, mapping) in enumerate(county_fields)},
+            "state": {field: (k, mapping) for k, (field, mapping) in enumerate(state_fields)},
+        }
+
+    def files_to_submissions(self):
+        """
+        Reads CSV files and converts the data to Sheepdog records
+        """
+        urls = {
+            "county": "https://raw.githubusercontent.com/covidcaremap/covid19-healthsystemcapacity/master/data/published/us_healthcare_capacity-county-CovidCareMap.csv",
+            "state": "https://raw.githubusercontent.com/covidcaremap/covid19-healthsystemcapacity/master/data/published/us_healthcare_capacity-state-CovidCareMap.csv",
+        }
+
+        for k, url in urls.items():
+            self.parse_file(url, csv_type=k)
+
+    def parse_file(self, url, csv_type):
+        print("Getting data from {}".format(url))
+        with closing(requests.get(url, stream=True)) as r:
+            f = (line.decode("utf-8") for line in r.iter_lines())
+            reader = csv.reader(f, delimiter=",", quotechar='"')
+
+            headers = next(reader)
+
+            if headers[0] == "404: Not Found":
+                print("  Unable to get file contents, received {}.".format(headers))
+                return
```

Comment:
should it error?

Reply:
I have taken that from JHU: https://github.com/uc-cdis/covid19-tools/blob/master/covid19-etl/etl/jhu.py#L177-L179

Reply:
for JHU there was 1 link that didn't have data yet so it was expected

but here (and in other ETLs) i think it should error

Reply:
it should also error in JHU now actually